popmon/visualization/utils.py

```diff
@@ -386,18 +370,50 @@ def xtick(lab):
                     lab = lab[:17] + "..."
```

Comment:
Could you move this xtick function out of this (and take top as parameter)? The version in plot_heatmap_b64 is actually correct.